e2e/e2e-scenario.sh

```diff
@@ -75,7 +75,7 @@ az vmss create -n ${VMSS_NAME} \
     --vm-sku $VM_SKU \
     --instance-count 1 \
     --assign-identity $msiResourceID \
-    --image "microsoft-aks:aks:aks-ubuntu-1804-2022-q1:2022.02.01" \
```

Comment:
Updating this image, since it needs the newer VHD (with the updated apt packages).

Reply:
Otherwise Agentbaker e2e will fail, even though the behavior is correct

Reply:
Will likely change it to a different image before merging (GPU tests seem to have restrictions due to it too)